project/apps/blog/src/app/dto/get-comments.dto.ts

```diff
@@ -0,0 +1,20 @@
+import { IsInt, IsOptional, Max, Min } from 'class-validator';
+import {
+  DEFAULT_COMMENTS_PAGE,
+  MIN_COMMENTS_LIMIT,
+  MAX_COMMENTS_LIMIT,
+  DEFAULT_COMMENTS_LIMIT,
+} from '@project/core';
+
+export class GetCommentsDto {
+  @IsOptional()
+  @IsInt()
+  @Min(DEFAULT_COMMENTS_PAGE)
+  page?: number = DEFAULT_COMMENTS_PAGE;
+
+  @IsOptional()
+  @IsInt()
+  @Min(MIN_COMMENTS_LIMIT)
+  @Max(MAX_COMMENTS_LIMIT)
+  limit?: number = DEFAULT_COMMENTS_LIMIT;
```

Comment:
Они всегда будут, тк засетаются либо пришедшие, либо по умолчанию
```suggestion
export class GetCommentsDto {
  @IsOptional()
  @IsInt()
  @Min(DEFAULT_COMMENTS_PAGE)
  page: number = DEFAULT_COMMENTS_PAGE;

  @IsOptional()
  @IsInt()
  @Min(MIN_COMMENTS_LIMIT)
  @Max(MAX_COMMENTS_LIMIT)
  limit: number = DEFAULT_COMMENTS_LIMIT;
```